File: christmas/december task 2/array tasks other languages/maximum.py
def maximumIn(array, largest):

    largest  = array[0]
        
    for count in range (1, len(array)):

        if(array[count] > largest ):

            largest = array[count];    

    return largest;
        
   

array = [9, 7, 2, 1];

largest = 0;

print(maximumIn(array, largest));

File: christmas/december task 2/array tasks other languages/Maximum.js

function maximumIn(array, largest) {

        largest  = array[0];
        
        

    for(count = 1; count < array.length; count ++) {
        if(array[count] > largest ) {

            largest = array[count]; 
            
        }

           
        }

        return largest;
        
   }

let  array = [9, 7, 2, 1];

largest = 0;

console.log(maximumIn(array, largest));
